cmd/skywire-cli/commands/node/gen-config.go

```diff
@@ -186,3 +189,19 @@ func defaultSkyproxyClientConfig() visor.AppConfig {
 		Port:      routing.Port(skyenv.SkyproxyClientPort),
 	}
 }
+
+func getLocalIPAddress() string {
+	addrs, err := net.InterfaceAddrs()
+	if err != nil {
+		log.WithError(err)
+	}
+
+	for _, a := range addrs {
+		if ipnet, ok := a.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
+			if ipnet.IP.To4() != nil {
```

Comment:
I'm not sure if this implementation is correct, but it looks like it works with IPv6. Probably this line should look like this:
```suggestion
			if ipnet.IP.To4() != nil || ipnet.IP.To16() != nil {
```
However, the suggestion needs to be tested.

Reply:
It does not matter to me whether we use IPv4 or IPv6 as long as we are using only one of these by default consitently, imo.